ores/score_processors/score_processor.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, scoring_contexts, score_cache=None,
         self.metrics_collector = metrics_collector or Null()
 
     def score(self, context, model, rev_ids, caches=None,
-              include_features=False, precache=False):
+              include_features=False, precache=False, features_cache={}):
```

Comment:
Yeah.  By the time we get to _process, it is processing an individual revision.  When you see `cache` it's the cache for an individual revision scoring.  When you see `caches`, it's a mapping of `rev_id` --> `cache`.